tests/test_serialize.py

```diff
@@ -1073,22 +1072,21 @@ def quadruple_x_plus_one(self) -> Annotated[int, PlainSerializer(lambda v: v + 1
     }
 
 
-def test_computed_field_custom_serializer_bad_signature():
-    error_msg = 'field_serializer on computed_field does not use info signature'
-
-    with pytest.raises(PydanticUserError, match=error_msg):
+def test_computed_field_custom_serializer_signature_with_info():
+    class Model(BaseModel):
+        x: int
 
-        class Model(BaseModel):
-            x: int
+        @computed_field
+        @property
+        def two_x(self) -> int:
+            return self.x * 2
 
-            @computed_field
-            @property
-            def two_x(self) -> int:
-                return self.x * 2
+        @field_serializer('two_x')
+        def ser_two_x_bad_signature(self, v, _info):
```

Comment:
Should probably rename this - doesn't have a bad signature now.

Reply:
Can probably delete this test honestly - not sure of the value here anymore, especially given the other test added above.